CHANGES.md

```diff
@@ -1,3 +1,9 @@
+## pending
+
+Light Client:
+
+- Expose latest_trusted from Supervisor Handle ([#394])
```

Comment:
Did we decide to have the light client changelog here or in the separate crate? 
@brapse 

Probably the changes related to the light client need a more general description and intro in the changelog but this is good for now 👌

Reply:
Just looked for precedence and there were entries for `Light Client` further down.